lang/en/quizaccess_seb.php

```diff
@@ -95,6 +95,30 @@
 
 // Capabilities.
 $string['seb:bypassseb'] = 'Bypass the requirement to view quiz in Safe Exam Browser.';
+$string['managetemplates'] = 'Manage Safe Exam Browser configuration templates.';
+$string['seb:manage_seb_requiresafeexambrowser'] = 'Modify Safe Exam Browser quiz setting: Require Safe Exam Browser';
+$string['seb:manage_seb_sebconfigtemplate'] = 'Modify Safe Exam Browser quiz setting: Select SEB template.';
+$string['seb:manage_seb_sebconfigfile'] = 'Modify Safe Exam Browser quiz setting: Select SEB config file.';
+$string['seb:manage_seb_suppresssebdownloadlink'] = 'Modify Safe Exam Browser quiz setting: Suppress download link.';
+$string['seb:manage_seb_linkquitseb'] = 'Modify Safe Exam Browser quiz setting: Quit link.';
+$string['seb:manage_seb_userconfirmquit'] = 'Modify Safe Exam Browser quiz setting: Confirm on quit.';
+$string['seb:manage_seb_allowuserquitseb'] = 'Modify Safe Exam Browser quiz setting: Allow quit.';
+$string['seb:manage_seb_quitpassword'] = 'Modify Safe Exam Browser quiz setting: Quit password.';
+$string['seb:manage_seb_allowreloadinexam'] = 'Modify Safe Exam Browser quiz setting: Allow reload.';
+$string['seb:manage_seb_showsebtaskbar'] = 'Modify Safe Exam Browser quiz setting: Show task bar.';
+$string['seb:manage_seb_showreloadbutton'] = 'Modify Safe Exam Browser quiz setting: Show reload button.';
+$string['seb:manage_seb_showtime'] = 'Modify Safe Exam Browser quiz setting: Show time.';
+$string['seb:manage_seb_showkeyboardlayout'] = 'Modify Safe Exam Browser quiz setting: Show keyboard layout.';
+$string['seb:manage_seb_showwificontrol'] = 'Modify Safe Exam Browser quiz setting: Show wifi control.';
```

Comment:
On par with Issue #63, cleaning up wifi => Wi-Fi.
```suggestion
$string['seb:manage_seb_showwificontrol'] = 'Modify Safe Exam Browser quiz setting: Show Wi-Fi control.';
```